components/ResultsPage/ResultsBoxes.tsx

```diff
@@ -0,0 +1,208 @@
+          {root.gis?.eligibility?.detail.split('\n')[0] === ('Eligible' || 'Admissible') ? <li key={root.gis}>{tsln.gis}</li> : ''}
+          {root.allowance?.eligibility?.detail.split('\n')[0] === ('Eligible' || 'Admissible') ? <li key={root.allowance}>{tsln.alw}</li> : ''}
+          {root.afs?.eligibility?.detail.split('\n')[0] === ('Eligible' || 'Admissible') ? <li key={root.afs}>{tsln.afs}</li> : ''}
+        </ul>
+      </div>
+
+      {/* Your estimated monthly total */}
+
+      <h2 id="estimated" className="h2 mt-5"><Image src="/money.png" alt="dollar sign" width={30} height={30} />&nbsp; {tsln.resultsPage.yourEstimatedTotal} {numberToStringCurrency(root.summary.entitlementSum, locale)}</h2>
+
+      <div className='pl-12'>
+        {tsln.resultsPage.basedOnYourInfoTotal} {numberToStringCurrency(root.summary.entitlementSum, locale)}
+
+        <h3 className='my-6 font-semibold'>{tsln.resultsPage.header}</h3>
+
+        <table className="hidden md:block text-left">
+          <thead className="font-bold border border-[#DDDDDD] bg-[#EEEEEE]">
+            <tr>
+              <th>{tsln.resultsPage.tableHeader1}</th>
+              <th>{tsln.resultsPage.tableHeader2}</th>
+            </tr>
+          </thead>
+
+          <tbody className="align-top">
+            <ResultsTableRow
+              heading={tsln.oas}
+              data={root.oas}
+              locale={locale}
+              showEntitlement={!root.summary.zeroEntitlements}
+            />
+            <ResultsTableRow
+              heading={tsln.gis}
+              data={root.gis}
+              locale={locale}
+              showEntitlement={!root.summary.zeroEntitlements}
+            />
+            <ResultsTableRow
+              heading={tsln.alw}
+              data={root.allowance}
+              locale={locale}
+              showEntitlement={!root.summary.zeroEntitlements}
+            />
+            <ResultsTableRow
+              heading={tsln.afs}
+              data={root.afs}
+              locale={locale}
+              showEntitlement={!root.summary.zeroEntitlements}
+            />
+            {!root.summary.zeroEntitlements && (
+              <tr className="border border-[#DDDDDD]">
+                <td>{tsln.resultsPage.tableTotalAmount}</td>
+                <td className="text-right min-w-[68px]">
+                  {numberToStringCurrency(root.summary.entitlementSum, locale)}
+                </td>
+              </tr>
+            )}
+          </tbody>
+        </table>
+      </div>
+
+      {/* Next steps for benefits you may be eligible */}
+
+      <hr className='my-12 border border-[#BBBFC5]' />
+
+      <h2 id="next" className="h2 mt-5">{tsln.resultsPage.nextSteps}</h2>
+
+      <MessageBox title={tsln.oas} eligible={true} eligibleText="Eligible" links={[{icon:'info', url:'canada.ca', alt:'info', text:"Learn more about OAS" }]}> Based en what you told us <strong>you do not need to apply</strong> You will ...</MessageBox>
```

Comment:
all strings should use translation:
 - "eligible" 
 - "info" 
 - "learn more about..."
 - "based on what you told us..."